models/cube.js

```diff
@@ -190,7 +200,7 @@ cubeSchema.index({
   numDecks: -1,
 });
 
-const Cube = mongoose.model('Cube', cubeSchema);
+const Cube = mongoose.model('Cube', withMigrations(cubeSchema, cubeMigrations));
```

Comment:
I don't like this pattern - I'd much rather have a one shot script to update the database